news-bundle/src/Resources/contao/modules/ModuleNews.php

```diff
@@ -229,6 +231,38 @@ protected function parseArticle($objArticle, $blnAddArchive=false, $strClass='',
 			$responseTagger->addTags(array('contao.db.tl_news.' . $objArticle->id));
 		}
 
+		// schema.org information
+		$objTemplate->getBasicSchemaOrgData = static function () use ($objTemplate, $objArticle): array
+		{
+			$jsonLd = array(
+				'@type' => 'NewsArticle',
+				'identifier' => '#/schema/news/' . $objArticle->id,
+				'url' => $objTemplate->link,
```

Comment:
This will be a relative URL such as `en\/news-detail\/new-contao-versions-in-short-intervalls.html`. It this correct or do we have to convert it to an absolute URL instead?

Reply:
I went for relative because we also do this for `ImageObject` currently. For schema.org it doesn't matter.